tests/h/util/user_test.py

```diff
@@ -11,5 +12,5 @@ def test_split_user():
 
 
 def test_split_user_no_match():
-    with pytest.raises(ValueError):
+    with pytest.raises(InvalidUserId):
         user_util.split_user("donkeys")
```

Comment:
I was amused by your `donkeys` and then realized, Lo!, it is my own `donkeys`. It is nice to amuse one's self.

Reply:
Amusing yourself is a great way of ensuring you both have the same sense of humour!

In my code submission for the interview I called my main accounting package 'h_ledger', which I'm fairly sure amused me an nobody else.